CMakeLists.txt

```diff
@@ -299,6 +299,8 @@ endif()
 
 option(ENABLE_VALGRIND "enables Valgrind memory testing in the test-suite" FALSE)
 
+option(ENABLE_MITRE "enables MITRE tests -- prone to random failures" FALSE)
```

Comment:
So, I think the failures are not random, but the tests require a very specific environment.